README.md

```diff
@@ -1,95 +1,108 @@
-## ChoiceScript Library (cslib)
+<h1><img src="./res/logo/cslib_logo%20flat.png" alt="C.S.Lib logo" width="43" align="left"/> ChoiceScript Library</h1>
 
-The ChoiceScript library (cslib) is a collection of common, re-usable routines (i.e. functions) for use in development of ChoiceScript games. 
+![Release](https://img.shields.io/github/release/ChoicescriptIDE/cslib) ![GitHub](https://img.shields.io/github/license/ChoicescriptIDE/cslib) ![GitHub all releases](https://img.shields.io/github/downloads/ChoicescriptIDE/cslib/total) ![Discourse topics](https://img.shields.io/discourse/topics?server=https%3A%2F%2Fforum.choiceofgames.com%2F) 
+[![Build Status](https://img.shields.io/endpoint.svg?url=https%3A%2F%2Factions-badge.atrox.dev%2FChoicescriptIDE%2Fcslib%2Fbadge%3Fref%3Dmain&style=flat)](https://actions-badge.atrox.dev/ChoicescriptIDE/cslib/goto?ref=main)
 
-cslib is written entirely in ChoiceScript itself, so is easy to understand, configure and use.
+The ChoiceScript Library (cslib) is a collection of common, re-usable routines (i.e. functions) for use in development of ChoiceScript games. It's an attempt to extend ChoiceScript's core functionalities using vanilla ChoiceScript only. It relies heavily on string parsing and string manipulation.
```

Comment:
"It relies heavily on string parsing and string manipulation." this sounds a bit scary and technical for the README. Part of the beauty of a library is you don't necessarily need to understand _how_ it's implemented (though it helps).